Updated GraphQL User Authentication

Dev-Dominic opened the pull request:
Changes:
- Updates graphql.rst docs to reflect the use of JSON Web Tokens access tokens.
- Updates made to purge legacy code, as well as, update naming convention of queries to be more neutral. `allDatapoints` to `datapoint`
- Updates test_graphql to reflect naming changes to queries

Note:
- Changes made can affect both client side applications for Pattoo, namely Pattoo-web and Pattoo-mobile.
- Client side projects need to be updated to match query name changes, as well as, store accessTokens and refresh tokens for continued use of GraphQL server on pattoo-api.

Resolves: #381

Dev-Dominic committed:
Purge legacy code

Legacy code of graphql queries from default graphene-Graphql package.
Changes made:

1. Omission of obscelte queries
2. Update new query names
3. Update graphql.rst docs to match current working queries

Dev-Dominic committed:
Update GraphQL Mutations for Authentication

Dev-Dominic committed:
Update test_graphql.py

gill876 reviewed (commented):
Have a successful Travis build

Dev-Dominic committed:
Merge branch '2020-Q4' of https://github.com/PalisadoesFoundation/pattoo

Dev-Dominic committed:
Add Refresh Token Mutation to graphql Docs

Dev-Dominic committed:
Merge branch 'master' of https://github.com/Dev-Dominic/pattoo

palisadoes commented on docs/graphql.rst:
```diff
@@ -181,10 +173,10 @@ The agentProgram value will be used later for getting a translation into a meani
     {
-      allAgent {
+      agent(token:
+      "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiYWRtaW4iOnRydWUsImp0aSI6IjcyMzFmZjk5LTE0NDktNDRhMS04YzE2LTY4OTMzNjgwZTU4YSIsImlhdCI6MTU5OTQyMjk5MywiZXhwIjoxNTk5NDI2NTkzfQ.UP1FSU3hNOI6EiEt2sMJ4V5n2BN3ttICsJFhJXJTowU"){
```
Trim the token length so that it fits within 80 characters. Easier readability. For example.

`eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiI... ...J4V5n2BN3ttICsJFhJXJTowU`

There are a number of repeats, so this would be an easy fix.

palisadoes reviewed (changes requested):
Trim the token length so that it fits within 80 characters. Easier readability. For example.

`eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiI... ...J4V5n2BN3ttICsJFhJXJTowU`

There are a number of repeats, so this would be an easy fix.

OK otherwise. It's a simple PR.

Dev-Dominic committed:
graphql doc update for #382

__Issue__: Lengthy lines for rst files.

__Fix__: Ensure most lines within graphql.rst was 80 characters long.
